src/backend/src/prisma-query-args/design-reviews.query-args.ts

```diff
@@ -0,0 +1,17 @@
+import { Prisma } from '@prisma/client';
+
+const designReviewQueryArgs = Prisma.validator<Prisma.Design_ReviewArgs>()({
+  include: {
+    userCreated: true,
+    teamType: true,
+    requiredMembers: true,
+    optionalMembers: true,
+    confirmedMembers: true,
+    deniedMembers: true,
+    attendees: true,
+    userDeleted: true,
+    wbsElement: true
+  }
+});
+
+export default designReviewQueryArgs;
```

Comment:
Looks like there's a no newline and end of file warning here, could you add a newline?